configs/eslint-config-ts/typescript.js

```diff
@@ -1,10 +1,27 @@
 /** @type {import('eslint').Linter.Config} */
 module.exports = {
+  root: true,
   extends: ['plugin:import/typescript', './noimport.js'],
   plugins: ['import'],
   settings: {
     'import/resolver': {
       typescript: {},
     },
   },
+  overrides: [
+    {
+      files: ['*.test-d.ts*', '*.test-d.ts*'],
+      rules: { 'import/no-unresolved': ['error', { ignore: ['tsd'] }] },
+    },
+  ],
+  rules: {
+    'sort-imports': ['error', { ignoreDeclarationSort: true }],
+    'import/order': [
+      'error',
+      {
+        'newlines-between': 'never',
+        alphabetize: { order: 'asc', caseInsensitive: true },
+      },
+    ],
+  },
```

Comment:
I added import, sort-import rules.
It will make us not to care import order and named import order. so we may concentrate core code.